{{cookiecutter.project_name}}/pipeline.yaml

```diff
@@ -327,6 +321,14 @@ Resources:
                         Effect: Allow
                         Action: 'ssm:GetParameters'
                         Resource: '*'
+                - PolicyName: CodeBuildSecretsManager
+                  PolicyDocument:
+                    Version: '2012-10-17'
+                    Statement:
+                      -
+                        Effect: Allow
+                        Action: 'secretsmanager:GetSecretValue'
+                        Resource: '*'
```

Comment:
Can we pass the specific ARN or a prefix instead of "*"?

Reply:
Agreed. Had to create a Parameter Store key/value for the 6 random character ARN suffix generated by Secrets Manager.